docs/getting-started/quick-start.md

```diff
@@ -4,12 +4,247 @@ sidebar_position: 2
 
 # Quick Start
 
+_Service Binding Operator should be [installed](./installing-service-binding.md) on the cluster prior to running this quick start guide._
+
 In this section we are providing a sample application that one can deploy and just use to play around.
 
-## K8S
+## Kubernetes
+
+### Application
+
+An application is a process running within a container. Examples include a NodeJS Express application, a Ruby Rails application or a Spring Boot application.
+
+Create a NodeJS application:
+
+```yaml
+kubectl apply -f - << EOD
+---
+apiVersion: apps/v1 ############################### GROUP/VERSION
+kind: Deployment    ############################### RESOURCE
+
+metadata:
+  name: nodejs-app  #################### NAME
+  labels:
+    app: nodejs
+    runtime: nodejs
+    runtime-version: 14-ubi7
+
+spec:
+  selector:
+    matchLabels:
+      app: nodejs
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: nodejs
+        deploymentconfig: nodejs
+    spec:
+      containers:
+      - name: nodejs
+        image: nodejs:latest
+        ports:
+        - containerPort: 8080
+          protocol: TCP
+EOD
+```
+
+Application can be a podSpec based resource like "Deployment" or "DeploymentConfig" or a non podSepc based resource like a Custom Resource Definition.
+
+### Service
+
+Apply the following CatalogSource:
+
+```yaml
+kubectl apply -f - << EOD
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+    name: sample-db-operators
+    namespace: openshift-marketplace
```

Comment:
this namespace exists only in OpenShift, but the section is about Kubernetes - from applied resource we also assume the presence of OLM?